FluentTerminal.App/Services/Implementation/DefaultValueProvider.cs

```diff
@@ -389,7 +398,39 @@ public IEnumerable<TerminalTheme> GetPreInstalledThemes()
                 }
             };
 
-            return new[] { defaultXterm, powerShell, homebrew, tomorrow };
+            var ubuntu = new TerminalTheme
+            {
+                Id = Guid.Parse("32519543-bf94-4db6-92a1-7bcec3966c82"),
+                Author = "Canonical",
+                Name = "Ubuntu Bash",
+                PreInstalled = true,
+                Colors = new TerminalColors
+                {
+                    Foregroun = "#EEEEEE",
+                    Background = "#300A24",
+                    Cursor = "#CFF5DB",
+                    CursorAccent = "#000000",
+                    Selection = "rgba(207, 245, 219, 0.99609375)",
```

Comment:
that's pretty much #f5f6f9. I think we ned to set alpha to something like 0.3-0.5 to make selected text still readable. This is a limitation of xterm.js, where we cannot set a selectedForeground color